### DISCUSSION TITLE
Add departure and arrival datetime

### DISCUSSION THREAD

**Discussion by khamaileon** [mthh/routingpy#115] [Ideas]
I'm thinking about adding departure_datetime and arrival_datetime to each Direction object. What do you think?

**Comment by nilsnolde** [comment]
I think that's fair. Then I'd also like to update Valhalla (not sure if GH or ORS have an open endpoint considering time, OSRM should have though). I frankly don't care about the commercial ones too much, but feel free to update those as well.
